src/Makefile.am

```diff
@@ -49,14 +49,59 @@ $(srcdir)/vmod_resolver.stub.c: vcc_dynamic_if.h
 @BUILD_VMOD_DYNAMIC@
 
 AM_TESTS_ENVIRONMENT = \
-	PATH="$(VMOD_TEST_PATH)" \
+	PATH="$(abs_builddir):$(VARNISH_TEST_PATH):$(PATH)" \
 	LD_LIBRARY_PATH="$(VARNISH_LIBRARY_PATH)"
 TEST_EXTENSIONS = .vtc
 VTC_LOG_COMPILER = varnishtest -vl
-AM_VTC_LOG_FLAGS = -Dvmod_dynamic="$(VMOD_DYNAMIC)"
+AM_VTC_LOG_FLAGS = \
+	-p vcl_path="$(abs_top_srcdir)/vcl:$(VARNISHAPI_VCLDIR)" \
+	-p vmod_path="$(abs_builddir)/.libs:$(vmoddir):$(VARNISHAPI_VMODDIR)"
+
+TESTS = \
```

Comment:
I would expect this to require some conditional on whether or not getdns is available